nx-dev/ui-home/src/lib/ci-for-monorepos.tsx

```diff
@@ -0,0 +1,469 @@
+import { Dialog, Transition } from '@headlessui/react';
+import { PlayIcon } from '@heroicons/react/24/outline';
+import { cx } from '@nx/nx-dev/ui-primitives';
+import { motion } from 'framer-motion';
+import Image from 'next/image';
+import Link from 'next/link';
+import { ComponentProps, Fragment, ReactNode, useState } from 'react';
+import {
+  ButtonLink,
+  SectionHeading,
+  Strong,
+  TextLink,
+} from '@nx/nx-dev/ui-common';
+import { AnimateValue, Marquee } from '@nx/nx-dev/ui-animations';
+import {
+  AzureDevOpsIcon,
+  BitbucketIcon,
+  GitHubIcon,
+  GitlabIcon,
+  JenkinsIcon,
+  TravisCiIcon,
+} from '@nx/nx-dev/ui-icons';
+
+export function CiForMonorepos(): JSX.Element {
+  return (
+    <section className="bg-slate-50 py-32 shadow-inner sm:py-40 dark:bg-slate-900">
+      <article className="mx-auto max-w-7xl px-6 lg:px-8">
+        <div className="max-w-5xl">
+          <SectionHeading
+            as="h2"
+            variant="title"
+            id="ci-for-monorepos"
+            className="scroll-mt-24"
+          >
+            Finally! CI that works for monorepos.
+          </SectionHeading>
+          <SectionHeading as="p" variant="subtitle" className="mt-6">
+            Current CI systems are <Strong>slow</Strong>,{' '}
+            <Strong>hard to maintain and unreliable</Strong>. With Nx Cloud we
```

Comment:
A comma missing, "cost efficient" and "task based" need hyphens. I also like replacing "new task-based approach" with "innovative task-based approach", to call out that we're the only ones doing it.

`With Nx Cloud,we introduce an innovative task based approach to making CI for monorepos not just fast, but also cost-efficient.`